.github/workflows/build.yml

```diff
@@ -118,6 +118,17 @@ jobs:
         # --no-scan:      For public fork PRs, we won't attempt to publish the scan
         run: |
           ./gradlew --build-cache --info $SCAN_ARG check -x test
+      - name: Check generated documentation
+        # Check if there are any empty files under ./site-docs/generated, If any empty files are found, print an error 
+        # message and list the empty files
+        run: |
+          ./gradlew clean siteDocTar
```

Comment:
It should not call `clean`, as doing so prevents subsequent steps from accessing the output of the previous build. Perhaps we should merge the `siteDocTar` task into the preceding step, and this step verify the contents of the generated documentation files.